applications/ShallowWaterApplication/python_scripts/benchmarks/base_benchmark_process.py

```diff
@@ -10,8 +10,13 @@ def Factory(settings, model):
     return BaseBenchmarkProcess(model, settings["Parameters"])
 
 class BaseBenchmarkProcess(KM.Process):
+    """The base class for the benchmarks."""
 
     def __init__(self, model, settings ):
+        """The constructor of the BaseBenchmarkProcess.
+
+        It is intended to be called from the constructor of deriving classes.
+        """
         super(BaseBenchmarkProcess, self).__init__()
```

Comment:
```suggestion
        super().__init__()
```

FYI now that we dropped support for python2 this is no longer needed

Reply:
thanks, I will remove this from all the python scripts on another PR